unittests/TestMain.cpp

```diff
@@ -7,21 +7,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "klee/Config/Version.h"
-
-#include "llvm/ADT/StringRef.h"
-#include "llvm/Support/Signals.h"
-
 #include "gtest/gtest.h"
 
 int main(int argc, char **argv) {
-
-#if LLVM_VERSION_CODE >= LLVM_VERSION(3, 9)
-  llvm::sys::PrintStackTraceOnErrorSignal(argv[0], true);
```

Comment:
Sorry, I should have stated that I want to keep this line (and the two corresponding includes).

Reply:
So this basically?

```C
#include "klee/Config/Version.h"

#include "llvm/ADT/StringRef.h"
#include "llvm/Support/Signals.h"

#include "gtest/gtest.h"

int main(...) {
  llvm::sys::PrintStackTraceOnErrorSignal(argv[0], true);
  testing::InitGoogleTest(&argc, argv);
  return RUN_ALL_TESTS();
}
```

Reply:
Except for the Version.h include (as we are not checking any version anymore), yes.